test/unit/helpers/httpApi.js

```diff
@@ -0,0 +1,488 @@
+var chai = require('chai');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var _ = require('underscore');
+
+var httpApi = require('../../../helpers/httpApi');
+
+describe('httpApi', function () {
+	describe('middleware', function () {
+		describe('cors', function () {
+			it('headers are set correctly', function () {
```

Comment:
Please stick to more descriptive way of arranging the tests. When you will read the message after, messages like
```
middleware
   cors
      headers are set correctly
```
or 
```
errorLogger
   error = null
```
doesn't really show what is happening in the test.
Message `should set Access-Control-Allow-Origin header to "*"` describes better what's going on. Also provide 2 separate checks for 2 checked headers